x11-terms/aterm/aterm-1.0.1-r5.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic eutils
+
+DESCRIPTION="rxvt compatible terminal emulator with transparency support"
+HOMEPAGE="http://aterm.sourceforge.net"
+SRC_URI="ftp://ftp.afterstep.org/apps/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE="background cjk xgetdefault"
+
+RDEPEND="
+	virtual/jpeg:0
+	media-libs/libpng:0=
+	background? ( media-libs/libafterimage )
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libICE
+"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto
+	x11-libs/libXt
```

Comment:
libXt is really DEPEND only?

Reply:
Apparently yes. Just needs some define to set some other define. Very awkward.